packages/ui/src/Input/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { HTMLAttributes } from "react";
+
+interface IInputProperties extends HTMLAttributes<HTMLInputElement> {
+  error?: boolean;
+  errorMessage?: string;
+  label?: string;
+  name?: string;
+  type?: "text" | "number";
+}
+
+export const Input = ({
+  error,
+  errorMessage,
+  label,
+  name,
+  onChange,
+  placeholder,
+  type,
+  ...others
+}: IInputProperties) => {
+  return (
+    <div className="input-field-wrapper">
```

Comment:
I think we will be using the same structure for all other form components. Could we use some generic classname here eg `field-wrapper` so that we can later have same css for all form components? You can put specific class along with it if you need. eg `field-wrapper input`.